fuzz_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notation
+
+import (
+	"testing"
+
+	orasRegistry "oras.land/oras-go/v2/registry"
```

Comment:
nit: by golang convention
```suggestion
	orasregistry "oras.land/oras-go/v2/registry"
```

Reply:
Done in https://github.com/notaryproject/notation-go/pull/460/commits/151d6aa45c9b23c3d61deeba61d11fa0d09b91ab